### PR TITLE
Update 0.46.0 release note

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
https://github.com/eclipse-openj9/openj9-docs/issues/1330

Updated the draft release note that includes the known issues related to -XX:+ShareOrphans

[skip ci]
Signed-off-by: Sreekala Gopakumar sreekala.gopakumar@ibm.com

### CODE REVIEW

File: doc/release-notes/0.46/0.46.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk22/pull/42">#42</a>, <a href="https://github.com/ibmruntimes/openj9-openjdk-jdk21/pull/140">#140</a>, <a href="https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/337">#337</a>, <a href="https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/743">#743</a>, <a href="https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/766">#766</a></td>
```
Comment (pshipton) [inline doc/release-notes/0.46/0.46.md:51]: None of these changes are added in this 0.46 release, they should be removed from this table. Related documentation for these should be in the user guide, a migration guide, or they can be moved to known issues. The extended Hot Code Replace deprecation notice should remain in known issues.

Comment (pshipton) [reply doc/release-notes/0.46/0.46.md:51]: Sorry, never mind.

File: doc/release-notes/0.46/0.46.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+<th valign="bottom">Impact</th>
+<th valign="bottom">Workaround</th>
+</tr>
+</thead>
+
+<tbody>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/19833">#19833</a></td>
+<td valign="top">The class comparison might not detect the removal of method access modifiers. For example, a change of a method from public to package-private.</td>
```
Comment (pshipton) [inline doc/release-notes/0.46/0.46.md:102]: This needs to be more specific, i.e. "Class comparison for the shared cache might not ..."

File: doc/release-notes/0.46/0.46.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/13767">#13767</a></td>
+<td valign="top">Compressed references mode is not available.</td>
+<td valign="top">Apple silicon macOS</td>
+<td valign="top">You can use only the large heap (non-compressed references) mode.</td>
+<td valign="top">None</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/60">#60</a></td>
```
Comment (pshipton) [inline doc/release-notes/0.46/0.46.md:144]: This problem is resolved in the 0.46 release, the note should be removed.

File: doc/release-notes/0.46/0.46.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+</tr>
+</thead>
+
+<tbody>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/pull/19833">#19833</a></td>
+<td valign="top">The class comparison might not detect the removal of method access modifiers. For example, a change of a method from public to package-private.</td>
+<td valign="top">All</td>
+<td valign="top">If you recompiled classes after removing the method access modifier and reran with the <tt>-XX:+ShareOrphans</tt> option, the old version of the class might still be used.</td>
```
Comment (hangshao0) [inline doc/release-notes/0.46/0.46.md:104]: > If you recompiled classes after removing the method access modifier and reran with the `-XX:+ShareOrphans` option

This is more accurate:
"If you run your application with `-XX:+ShareOrphans` first, then recompiled classes after removing the method access modifier and reran with the `-XX:+ShareOrphans` option"

Comment (Sreekala-Gopakumar) [reply doc/release-notes/0.46/0.46.md]: Done

**Comment by Sreekala-Gopakumar** [comment]
@hangshao0 - I have incorporated all the suggested changes. Please check and confirm. And if everything is ok, could you please merge the changes. Thanks!

**Review by hangshao0 — APPROVED** [review]

**Review by pshipton — APPROVED** [review]

**Pull request merged**